src/change_stream/options.rs

```diff
@@ -66,10 +68,26 @@ pub struct ChangeStreamOptions {
     #[builder(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub start_after: Option<ResumeToken>,
+
+    /// The criteria used to select a server for this operation.
+    ///
+    /// If none is specified, the selection criteria defined on the object executing this operation
+    /// will be used.
+    #[builder(default)]
+    #[serde(skip)]
+    #[serde(rename = "readPreference")]
+    pub selection_criteria: Option<SelectionCriteria>,
+
+    /// Specifies the default read concern for operations performed on the Client. See the
+    /// ReadConcern type documentation for more details.
```

Comment:
Can we wrap `Client` and `ReadConcern` in backticks to signal they're types?

Reply:
yes, fixed

Reply:
The main portion of the docstring doesn't really apply here--it looks like it's the same text that was used for `read_concern` on `ClientOptions`. We should rewrite this to be specific to change streams, e.g. "The read concern to be used by the `ChangeStream`."

Also, can we add a newline between the opening paragraph and the subsequent lines about which levels are supported?

Reply:
fixed